projects/bp-gallery/.eslintignore

```diff
@@ -2,3 +2,4 @@ node-modules
 src/reportWebVitals.js
 src/react-app-env.d.ts
 src/graphql/schema/loadRemoteSchema.js
+src/prototypes/demo/Demo.tsx
```

Comment:
Is this intended? If so, why?

Reply:
random linter warnings and errors that couldn't be auto-fixed and I couldn't be bothered to manually fix a demo prototype that we aren't using anyways